Snowplow iOSTests/TestEvents.m

```diff
@@ -22,18 +22,6 @@ - (void)testEvent {
     event.contexts = self.goodCustomContext;
     XCTAssertNotNil(event);
     event = nil;
```

Comment:
This line is probably no longer necessary. Also not clear why the check is no longer necessary, but I'm sure there is a reason. Finally, does the test still make sense? It just checks that event is not nil after initiating an instance of it. Seems not very useful.

Reply:
I think you are right. The test doesn't have much sense because SPPageView constructor can only raise an exception.
However, in ObjC the constructor could return nil (even if it's rarely used) so checking that the object is NotNil is not a mistake.

In this case, I removed half of the test because it was used to check if the event is able to raise exception when an added context is not a `SelfDescribingJson`.
That was needed because the tracker internally has a lot of raw types. In this case, I set the generic type for contexts as SelfDescribingJson, so, in theory, the compiler should warn the developer if they assign a wrong type to a context array element. For this reason this test should be redundant.